src/config/src/meta/dashboards/v2/mod.rs

```diff
@@ -19,7 +19,9 @@ use utoipa::ToSchema;
 
 use crate::meta::stream::StreamType;
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema)]
+type OrdF64 = ordered_float::OrderedFloat<f64>;
```

Comment:
`OrderedFloat` is a wrapper type around `f64` that implements `Eq` `Org` and `Hash`, enabling structs with `OrdF64` fields to derive those traits as well. Normally structs with `f64` fields cannot derive those traits since `f64` does not implement those traits.

Reply:
I was wondering if we could move this to the mod level, instead of defining it multiple times in different versions. Think that's easier for any changes in the future.

Reply:
Updated